### PR TITLE
HPCC-17186 DOCS: Spray queue dropdown

### PR DESCRIPTION
**Pull request by g-pan**
Fix HPCC-17186 DOCS: Spray queue dropdown
Document a new queue select field inside of the spray areas of the landing zone widget.
@JamesDeFabia please review
@miguelvazq please review, verify if this sufficiently covers the new queues

Signed-off-by: G-Pan <greg.panagiotatos@lexisnexis.com>

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [x] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [x] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browers
  - [ ] The component(s) render as expected

### CODE REVIEW

**Comment by miguelvazq** [comment]
@g-pan The queue screenshot is not included here.

File: docs/ECLWatch/ECLWa_mods/ECLWatchSrc.xml [hunk 1]
```diff
@@ -1633,10 +1633,11 @@
           </mediaobject>
         </figure></para>
 
-      <para>Click on the arrow next to a drop zone container to expand. The
-      files on the drop zone display. You can choose to upload, download, or
-      delete any files on the drop zone using the landing zone action buttons.
-      You can also spray files to a cluster from this page.</para>
+      <para>Click on the arrow next to a drop zone container, or folder to
```
Comment (JamesDeFabia) [inline docs/ECLWatch/ECLWa_mods/ECLWatchSrc.xml]: comma not needed -- "container or folder"

**Review by JamesDeFabia — CHANGES_REQUESTED** [review]
The title says Spray queue dropdown, but I don't see that described or depicted.

**Comment by richardkchapman** [comment]
@g-pan Back to you

**Comment by g-pan** [comment]
@JamesDeFabia please review
@miguelvazq please review

**Comment by JamesDeFabia** [comment]
Looks good from my point of view.  @miguelvazq You get the last word.

**Review by miguelvazq — COMMENTED** [review]
Seems that ECLWA412.jpg and ECLWA440.jpg are different themes of ECLWatch. 440 is using a theme that we are going to use in a later release and 412 is a different theme altogether (perhaps a different browser)

**Comment by richardkchapman** [comment]
@g-pan Back to you

**Comment by g-pan** [comment]
updated. @JamesDeFabia please review @miguelvazq please review

**Comment by JamesDeFabia** [comment]
Looks good from my point of view. @miguelvazq  Please give final approval

**Comment by miguelvazq** [comment]
@g-pan Not sure if it matter but, those internal IP's are not blurred. Not sure what you do documentation wide.

**Comment by g-pan** [comment]
Should be ok, still better safe than sorry. Good catch. Image updated.

**Comment by miguelvazq** [comment]
Looks good now.

**Pull request merged**